scrapers/utils.py

```diff
@@ -2,36 +2,38 @@
 import errno
 import hashlib
 import inspect
-import requests
+import re
 import shutil
 import sys
 import tempfile
 from contextlib import contextmanager
 from datetime import datetime
 from pathlib import Path
 
+import requests
+
 
 @contextmanager
-def download_filepath(scraper_filepath):
+def download_directory(scraper_filepath):
     """
-    Context manager to create a file for the scraper to write the content to
+    Context manager to create a directory for the scraper to store downloaded files.
     arg `scraper_filepath` should always be the filepath of the scraper file
     (i.e. `__file__`)
 
     Based on: https://stackoverflow.com/a/22726782
 
     Example:
-    with download_filepath(__file__) as filepath:
+    with download_directory(__file__) as directory:
+        filename = get_standardized_filename(__file__, id)
```

Comment:
can we not assume this is going to be an id in the second argument

```suggestion
        filename = get_standardized_filename(__file__, link)
```